modules/twinklexfd.js

```diff
@@ -1076,21 +1244,24 @@ Twinkle.xfd.callbacks = {
 			var initialContrib = pageobj.getCreator();
 			var params = pageobj.getCallbackParameters();
 
+			// Also notify the user who owns the subpage if they are not the creator
+			params.userspaceOwner = mw.config.get('wgTitle').indexOf('/') === -1 ? mw.config.get('wgTitle') : mw.config.get('wgTitle').substring(0, mw.config.get('wgTitle').indexOf('/'));
+			if (params.notifyuserspace) {
+				if (params.userspaceOwner !== initialContrib) {
+					Twinkle.xfd.callbacks.mfd.userNotificationMain(params, params.userspaceOwner, 'Notifying owner of userspace');
+				}
+			}
+
 			// Disallow warning yourself
 			if (initialContrib === mw.config.get('wgUserName')) {
 				pageobj.getStatusElement().warn('You (' + initialContrib + ') created this page; skipping user notification');
 			} else {
+				// Used to ensure we only add to the userspace
+				// log once, after notifying the initial creator
+				params.initialContrib = initialContrib;
 				// Really notify the creator
 				Twinkle.xfd.callbacks.mfd.userNotificationMain(params, initialContrib, 'Notifying initial contributor');
 			}
-
-			// Also notify the user who owns the subpage if they are not the creator
-			if (params.notifyuserspace) {
-				var userspaceOwner = mw.config.get('wgTitle').indexOf('/') === -1 ? mw.config.get('wgTitle') : mw.config.get('wgTitle').substring(0, mw.config.get('wgTitle').indexOf('/'));
-				if (userspaceOwner !== initialContrib) {
-					Twinkle.xfd.callbacks.mfd.userNotificationMain(params, userspaceOwner, 'Notifying owner of userspace');
-				}
-			}
 		},
 		userNotificationMain: function(params, initialContrib, actionName) {
```

Comment:
parameter name `initialContrib` should be changed (maybe to `user`) as it could refer to the initial contributor or userspace owner.

Reply:
Went for `userTarget`, but yeah, that'd be clearer.